antora-playbook.yml

```diff
@@ -124,18 +123,8 @@ content:
   # - url: https://git@github.com/couchbase/docs-cloud-native
   - url: https://github.com/couchbase/docs-cloud-native
     branches: [cloud-native-2.2]
-#  - url: https://github.com/couchbaselabs/tutorials
-  - url: https://github.com/couchbase/developer-content
-  - url: https://github.com/couchbaselabs/tutorial-template
-  - url: https://github.com/couchbaselabs/mobile-travel-sample
-    branches: [master]
-    start_path: content
   # - url: https://github.com/couchbaselabs/hotel-finder-react-native
   # - url: https://github.com/couchbaselabs/hotel-lister-cordova
-#  - url: https://github.com/couchbaselabs/tutorials-contrib
-  - url: https://github.com/couchbaselabs/mobile-training-todo
-    branches: tutorials
-    start_path: content
   - url: https://github.com/couchbaselabs/UniversityLister-Android
```

Comment:
While you're here, I think the following modules are all tutorial content too.

And looking at the names vaguely reminds me of a chat a while back with @RichardSmedley... didn't you find some reason for which this content was spared, as it's still required by PM / not yet adopted by Dev Advocacy?

Reply:
You may be right, but I can't remember - DA are at SKO this week, so we may not get a quick answer on that.